packages/react/src/components/Tabs/next/Tabs.js

```diff
@@ -242,6 +245,10 @@ TabList.propTypes = {
    */
 
   contained: PropTypes.bool,
+  /**
+   * If using `IconTab`, specify the size of the icon being used.
+   */
+  iconSize: PropTypes.oneOf(['16', '20']),
```

Comment:
```suggestion
  iconSize: PropTypes.oneOf(['default', 'lg']),
```

I believe we should use the size names from other components to match convention there, let me know if I'm off 👍